Fabric.Authorization.IntegrationTests/SqlServer/SqlServerClientTests.cs

```diff
@@ -7,7 +7,7 @@ namespace Fabric.Authorization.IntegrationTests.SqlServer
     [Collection("SqlServerTests")]
     public class SqlServerClientTests //: ClientTests
     {
-        public SqlServerClientTests(IntegrationTestsFixture fixture)// : base(fixture,StorageProviders.SqlServer)
```

Comment:
Should this still be commented out?

Reply:
yeah because we dont want them running on the build server yet